src/main/java/ch/njol/skript/effects/EffContinue.java

```diff
@@ -61,39 +58,36 @@ public class EffContinue extends Effect {
 	static {
 		Skript.registerEffect(EffContinue.class,
 			"continue [this loop|[the] [current] loop]",
-			"continue [the] %*integer%(st|nd|rd|th) loop"
+			"continue [the] <[1-9][0-9]*>(st|nd|rd|th) loop"
```

Comment:
We need to consider that this will not work for any updates to the number parsing (e.g. underscores). Maybe we can pull from the pattern used by the classinfo?